researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -47,27 +47,23 @@
 public abstract class BridgeDataProvider extends DataProvider {
     private static final Logger logger = LoggerFactory.getLogger(BridgeDataProvider.class);
 
-    protected final Gson gson = RestUtils.GSON;
-
     // set in initialize
-    protected UserLocalStorage userLocalStorage;
-    protected ConsentLocalStorage consentLocalStorage;
     protected TaskHelper taskHelper;
     protected UploadHandler uploadHandler;
 
+    private final StorageAccessWrapper storageAccessWrapper;
```

Comment:
This was created prior to having more ability to refactor ResearchStack to isolate dependencies that force a call to Android APIs, which all throw RuntimeException("Stub!") without Robolectric (which is much slower than normal unit tests)

StorageAccess performs Android calls on class load, so if you call  mock/createMock(StorageAccess.class), it will barf on this line (IIRC):
    private Handler handler = new Handler(Looper.getMainLooper());